React/Modules/RCTAsyncLocalStorage.m

```diff
@@ -67,7 +67,11 @@ static void RCTAppendError(NSDictionary *error, NSMutableArray<NSDictionary *> *
   static NSString *storageDirectory = nil;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
+#if TARGET_OS_TV
+    storageDirectory = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES).firstObject;
```

Comment:
No there doesn't seem to be.  For persistent storage, apps can use NSUserDefaults (for tiny amounts of data) or iCloud sync.  https://developer.apple.com/library/tvos/documentation/General/Conceptual/AppleTV_PG/